src/Data/List.purs

```diff
@@ -180,6 +200,15 @@ manyRec p = tailRecM go Nil
     aa <- (Loop <$> p) <|> pure (Done unit)
     pure $ bimap (_ : acc) (\_ -> reverse acc) aa
 
+-- Questionable whether this should be specialized
+-- -- | Create a list containing a value repeated n times
+-- replicate :: forall a. Int -> a -> List a
+-- replicate num x = go num Nil
+--   where
+--   go n xs | n < 1 = xs
+--           | otherwise = go (n - 1) (x : xs)
```

Comment:
I'm wondering whether this should be added. Does anyone use this? Would anyone use this?

Reply:
I'm wondering the same. Current plan is to just provide a specialized `replicate` for `List.Lazy`, and otherwise use `Unfoldable`'s `replicate` here. So I'd remove this commented block during clean-up.
The full API proposal can be found in https://github.com/purescript/purescript-lists/issues/183#issuecomment-825460643. Some related questions about what to include or not can be found at the end of that comment.